package.json

```diff
@@ -272,6 +276,7 @@
 		"test": "npm run lint && npm run test-unit",
 		"test:create-block": "./bin/test-create-block.sh",
 		"test-e2e": "wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
+		"test-e2e:playwright": "playwright test --config packages/e2e-tests-playwright/playwright.config.ts",
```

Comment:
Maybe we'll need a script in `@wordpress/scripts` too, but right now it's just a standard `@playwright/test` command.